akka-http-core/src/main/resources/reference.conf

```diff
@@ -268,6 +268,12 @@ akka.http {
       # You can use value `1` to create a strict entity if the request contains at most a single data frame, and a streamed
       # entity otherwise.
       #
+      # If `min-collect-strict-entity-size > 0` and a request is cancelled (RST_STREAM) before enough data has been collected and the request
+      # is dispatched, then the request is silently discarded.
+      #
+      # To avoid flow control dead-locks, the value must be both smaller or equal than `incoming-stream-level-buffer-size`
```

Comment:
With that extra condition, I weaseled myself out of the requirement to add code to track stream level windows for streams that are currently collecting entity data.

It also makes sense because I'd like to keep the invariant that a connection never buffers more data than the smaller of `incoming-stream-level-buffer-size * max-concurrent-streams` or `incoming-connection-level-buffer-size`.